Src/WindowsServer/WindowsServer.Shared/Implementation/AppServiceEnvVarMonitor.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.Implementation
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Utility to monitor the value of environment variables which may change 
+    /// during the run of an application. Checks the environment variables 
+    /// intermittently.
+    /// </summary>
+    internal static class AppServiceEnvVarMonitor
+    {
+        internal static TimeSpan CheckInterval = TimeSpan.FromSeconds(30);
+        internal static DateTime LastCheckTime = DateTime.MinValue;
+
+        /// <summary>
+        /// Environment variables tracked by this monitor.
+        /// </summary>
+        internal static Dictionary<string, string> CheckedValues = new Dictionary<string, string>()
+        {
+            { "WEBSITE_SITE_NAME", string.Empty },
+            { "WEBSITE_HOME_STAMPNAME", string.Empty },
+            {  "WEBSITE_HOSTNAME", string.Empty }
```

Comment:
Nit: extra space (alignment)

Reply:
Done, this was breaking the build (stylecop).